tests/forms/FormFieldTest.php

```diff
@@ -3,7 +3,52 @@
  * @package framework
  * @subpackage tests
  */
-class FormFieldTest extends SapphireTest implements TestOnly {
+class FormFieldTest extends SapphireTest {
+
+	public function testDefaultClasses() {
+		Config::nest();
+
+		Config::inst()->update('FormField', 'default_classes', array(
+			'class1',
+		));
+
+		$field = new FormField('MyField');
+
+		$this->assertContains('class1', $field->extraClass(), 'Default class not added to class list');
+
```

Comment:
@dhensby I agree with you: the negative version makes more sense than the positive when displayed in the error output, but every other core test is written in the positive sense (I was just pointing out “our” convention for those messages).

We’re [not alone](https://github.com/symfony/DependencyInjection/blob/master/Tests/ContainerTest.php#L99) in messages being written this way, but the wording on [PHPUnit assertion docs](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertArrayHasKey) always starts:

> Reports an error identified by $message... 

Our messages don’t really identify the error, so perhaps they should all be rewritten the other way around?